src/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1730,10 +1719,9 @@ public int getAlleleIndex(final Allele allele) {
      * @return a list of indices for each allele, in order
      */
     public List<Integer> getAlleleIndices(final Collection<Allele> alleles) {
-        final List<Integer> indices = new LinkedList<Integer>();
-        for ( final Allele allele : alleles )
-            indices.add(getAlleleIndex(allele));
-        return indices;
+        return alleles.stream()
+                .map(this::getAlleleIndex)
+                .collect(Collectors.toCollection(LinkedList::new));
```

Comment:
sure. I didn't want to change the result without understanding why it was a linked-list to begin with...